server/app/org/scalaexercises/exercises/controllers/ExercisesController.scala

```diff
@@ -126,22 +117,19 @@ class ExercisesController(
         def logError(method: String, mainMsg: String, ex: Option[Throwable] = None) = {
           val msg = s"[$method] $mainMsg. ExerciseEvaluation: $evaluation with user $user"
           ex match {
-            case Some(e) ⇒ Logger.error(msg, e)
-            case None    ⇒ Logger.error(msg)
+            case Some(e) ⇒ Logger(this.getClass).error(msg, e)
+            case None    ⇒ Logger(this.getClass).error(msg)
```

Comment:
We could initialize the logger as part of the class and we are not creating new instances every time we are using it